src/pages/VaccinePage.js

```diff
@@ -95,11 +97,13 @@ export const VaccinePageComponent = ({
   >
     <FlexRow style={{ justifyContent: 'flex-end', marginBottom: 10 }}>
       <PageButton text={buttonStrings.add_sensor} onPress={toNewSensorPage} />
-      <PageButton
-        text={navStrings.dispensary}
-        onPress={toDispensingPage}
-        style={{ marginLeft: 8 }}
-      />
+      {hasVaccines && (
```

Comment:
hiding the button if there are no vaccines to dispense - partly because step three is a bit rubbish if there are no vaccines and this works around the problem